test_scripts/API/VehicleData/StabilityControlsStatus/013_SubscribeVehicleData_StabilityControlsStatus_DISALLOWED_by_policies.lua

```diff
@@ -0,0 +1,52 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0253-New-vehicle-data-StabilityControlsStatus.md
+--
+-- Description: Check subscription on StabilityControlsStatus data which is disallowed by Policies
+--   via SubscribeVehicleData RPC
+--
+-- Precondition:
+-- 1) SDL and HMI are started
+-- 2) App is registered
+-- 3) PTU is successfully performed and StabilityControlsStatus is not allowed
+-- 4) App is activated
+
+-- Steps:
+-- 1) App sends SubscribeVehicleData request with (stabilityControlsStatus = true) to SDL
+-- SDL does not send VehicleInfo.SubscribeVehicleData request to HMI
+-- SDL sends SubscribeVehicleData response with (success: false, resultCode: "DISALLOWED",
+--   stabilityControlsStatus = { resultCode = "DISALLOWED", dataType = "VEHICLEDATA_STABILITYCONTROLSSTATUS"})
+-- 2) HMI sends VehicleInfo.OnVehicleData notification with StabilityControlsStatus data
+-- SDL does not send OnVehicleData notification with received from HMI data to App
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('test_scripts/API/VehicleData/commonVehicleData')
+
+--[[ Local variables ]]
+local vdNameDisallowed = "stabilityControlsStatus"
+local mobileRequest = {
+  [vdNameDisallowed] = true
+}
+local hmiRequest = {}
+local mobileResponse = { 
+  success = false,
+  resultCode = "DISALLOWED",
+  [vdNameDisallowed] = common.getSubscribeVehicleDataHmiResponse("DISALLOWED", vdNameDisallowed)
+}
+
+--[[ Scenario ]]
+common.Title("Preconditions")
+common.Step("Clean environment", common.preconditions)
+common.Step("Start SDL, init HMI, connect default mobile", common.start)
+common.Step("Register App", common.registerApp)
+common.Step("PTU", common.policyTableUpdate, { common.ptUpdateMin })
```

Comment:
@mvorobio Performed PTU disallows `stabilityControlsStatus` parameter not only for tested in this script RPC (SubscribeVehicleData). So the script can miss possible class of defects when policy works incorrect for new parameter.
I would propose to disable only new parameter for only tested RPC (`stabilityControlsStatus` parameter for SubscribeVehicleData)

This comment relates next script too.

Reply:
@aderiabin please check https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/15/commits/a5a63104ea8ee63778808bf575be76cbb9fb7918